CHANGELOG.md

```diff
@@ -1,2 +1,10 @@
-- Adds auth blocking triggers to the auth and identity namespaces (1080).
-- Add support for secrets for v2 triggers (#1079).
+- Add CPU option. Rip out deprecated apiVersion [pull request](https://github.com/firebase/firebase-functions/pull/1077)
```

Comment:
Can we change all of these to end with the PR number? It might look weird if we don't follow our own convention for prior releases.
```
- Add CPU option. Rip out deprecated apiVersion (#1077).
```

Reply:
Updated!